content/03.use-cases-a.md

```diff
@@ -50,9 +52,11 @@ GitHub supports a broad set of mechanisms for hosting educational materials.
 The entire process of running a course, workshop, or lecture, can all be done openly on GitHub including material development, web hosting, and delivery, and even submission and grading of assignments.
 While there are other purpose-built platforms for this, GitHub provides a free, open-source alternative.
 
-Making presentations, syllabi and other course materials can be done through most major high-level programming languages such as R, with `RMarkdown` [@isbn:9781138359338], Python, with `python-ppt` (<https://python-pptx.readthedocs.io>), and Julia, with `Remark.jl` (<https://juliapackages.com/p/remark>), and be version-controlled and stored in GitHub.
-Once content is made, hosting a course website can be done through GitHub Pages (<https://pages.github.com>; _e.g._, <https://github.com/topics/course-website>).
-This way, course content can be available to enrolled students, as well as a global pool of learners and teachers interested in the material.
-Content can then be delivered via the course website, and/or a GitHub Organization with, for example, template repositories for assignments.
-Student submissions are perhaps the most challenging component, but the new GitHub Classroom tool (<https://classroom.github.com>) allows instructors to host private assignments to be submitted collaboratively or individually as code or PDF files, and even build autograding tests.
+<!-- remove/simplify stuff about tools to make documents.  out of scope -->
+Syllabi, presentations, and other course materials can be version-controlled and stored in GitHub. 
+Although plain text formats such as markdown or HTML work best with GitHub and help to keep the total size of the repository within limits, any file type can be stored on and downloaded from GitHub including PowerPoint and Word documents.
+Content can then be delivered by sharing a link to the GitHub repository or creating a template repository with assignments that students can use.
+Using GitHub to host a course website is also an option for making course content more widely and easily accessible (see [Hosting a Website](#hosting-a-website)).
+
+Using GitHub to collect student submissions is perhaps the most challenging component, but the new GitHub Classroom tool (<https://classroom.github.com>) allows instructors to host private assignments to be submitted collaboratively or individually as code or PDF files, and even build autograding tests.
```

Comment:
maybe change "the most challenging component" to "a more challenging component"

Reply:
I added @fhillemann's suggestion in review comment below

Reply:
I think I'll go with something like "a more advanced use".